source/jsonnet/england-wales/census_household.jsonnet

```diff
@@ -173,7 +173,7 @@ function(region_code, census_month_year_date) {
         },
         {
           type: 'List',
-          for_list: 'visitors',
+          for_list: 'visitor',
```

Comment:
Just a point that 'visitor' (singular) doesn't seem to be the right name for this list - visitors seems more appropriate as there could be more than one. 'household' implies more than one person. It seems like a mistake that's been pulled in from the test_repeating_sections_with_hub_and_spoke schema.